pkg/phases/argocdoperator/install.go

```diff
@@ -0,0 +1,27 @@
+package argocdoperator
+
+import (
+	"github.com/flanksource/karina/pkg/platform"
+)
+
+const (
+	Namespace = "argocd"
+)
+
+func Deploy(platform *platform.Platform) error {
+	if platform.ArgoCDOperator.IsDisabled() {
+		if err := platform.DeleteSpecs("", "argocd-operator.yaml"); err != nil {
+			platform.Warnf("failed to delete specs: %v", err)
+		}
+		return nil
```

Comment:
Swallowing deployment failures should only be by exception, not by default
```suggestion
		return platform.DeleteSpecs("", "argocd-operator.yaml"); 
```

Reply:
I know doing this will propagate the exception to the function calling this Deploy function but I'm not sure I fully understand what you are saying. Can you explain more or give an example?

Reply:
We are having this same logic in postgresoperator/install.go, should we refactor that to make it consistent?

Reply:
Yes - that makes sense